File: PredioResidencial/src/main/java/com/acueducto/PredioResidencial/model/Request.java
package com.acueducto.PredioResidencial.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDateTime;

@AllArgsConstructor
@Getter
@Setter
@ToString
public class Request {

    private int id;
    private String propietario;
    private String direccion;
    private LocalDateTime fechaRegistro;
    private int estrato;
    private double consumo;
    private double valorFactura;
    private String tipoComercio;
    private String numeroLicenciaComercial;


}

File: Cliente/Cliente/Request.cs
namespace Cliente
{
    internal class Request
    {
        public int Id { get; set; }
        public string Propietario { get; set; }
        public string Direccion { get; set; }
        public string EstadoCuenta { get; set; }
        public int Estrato { get; set; }
        public double Consumo { get; set; }
        public double Subsidio { get; set; }
        public string TipoVivienda { get; set; }
    }
}